audits/manifest/short-name-length.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+
+class ManifestShortNameLength extends Audit {
+  /**
+   * @override
+   */
+  static get tags() {
+    return ['Manifest'];
+  }
+
+  /**
+   * @override
+   */
+  static get name() {
+    return 'manifest-short-name-length';
+  }
+
+  /**
+   * @override
+   */
+  static get description() {
+    return 'Manifest short_name won\'t be truncated';
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    let isShortNameShortEnough = false;
+    const manifest = artifacts.manifest.value;
+
+    if (manifest && manifest.short_name) {
+      // Historically, Chrome recommended 12 chars as the maximum length to prevent truncation.
+      isShortNameShortEnough = (manifest.short_name.value.length <= 12);
+    }
+
+    return ManifestShortNameLength.generateAuditResult(isShortNameShortEnough);
```

Comment:
return a `debugString` in the negative case? Or is the `auditDescription: false` clear enough on its own?